### PR TITLE
fix(login-approved-styling)

### PR DESCRIPTION
**Pull request by jjBlockchain**
Web4-2658

### CODE REVIEW

**Comment by Milan-Popich** [comment]
Approved styling? How about working on getting the wallet actually to sync up to the totals the customer owns? All this other work you are trying to do is absolutely irrelevant and it is a waste of time money and it is unbelievable that all of you can’t figure out the basics to make something function properly. Start looking for another job because I am creating AI that will replace all of you. Goal is in less than a year all human programming will be extinct like the dinosaurs. So perhaps you can become stylist.
Please refrain from sending me your stylist improvement when the entire application lacks the ability for the customer to access their money.
Min health
Milan Popich
Unstoppable Greatness

In Health,
Milan Popich

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
Op woensdag, september 18, 2019 9:24 AM, jjBlockchain <notifications@github.com> schreef:

**Comment by plondon** [comment]
> How about working on getting the wallet actually to sync up to the totals the customer owns

Can you open an issue instead of commenting on a PR?

File: packages/blockchain-wallet-v4-frontend/src/scenes/AuthorizeLogin/index.js [hunk 1]
```diff
@@ -57,14 +69,19 @@ class AuthorizeLogin extends React.PureComponent {
           value={value}
           onAccept={this.onAccept}
           onReject={this.onReject}
+          handleSuccessContainer={this.handleSuccessContainer}
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/scenes/AuthorizeLogin/index.js]: `handleSuccessContainer={() => this.setState({ loginApproved: true })}`

File: packages/blockchain-wallet-v4-frontend/src/scenes/AuthorizeLogin/index.js [hunk 2]
```diff
@@ -48,6 +56,10 @@ class AuthorizeLogin extends React.PureComponent {
     analyticsActions.logEvent(VERIFY_DEVICE_REJECTED)
   }
 
+  handleSuccessContainer (e) {
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/scenes/AuthorizeLogin/index.js]: remove and just inline the `setState`

File: packages/blockchain-wallet-v4-frontend/src/scenes/AuthorizeLogin/index.js [hunk 3]
```diff
@@ -19,12 +25,14 @@ const {
 class AuthorizeLogin extends React.PureComponent {
   constructor (props) {
     super(props)
+    this.handleSuccessContainer = this.handleSuccessContainer.bind(this)
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/scenes/AuthorizeLogin/index.js]: remove

**Comment by johnsonsan** [comment]
Can you keep me informed when need to implement

**Pull request merged**